dask/dataframe/tests/test_dataframe.py

```diff
@@ -1750,12 +1750,18 @@ def test_sorted_index_single_partition():
         df.set_index('x'))
 
 
-def test_info(capsys):
-    df = pd.DataFrame({'long_column_name': [1, 2, 3, 4], 'short_name': [1, 0, 1, 0]})
-    ddf = dd.from_pandas(df, npartitions=1)
-    ddf.info()
-    out, err = capsys.readouterr()
-    assert out == ("<class 'dask.dataframe.core.DataFrame'>\n"
-                  "Data columns (total 2 columns):\n"
-                  "long_column_name    int64\n"
-                  "short_name          int64\n")
+def test_info_dataframe():
+    from StringIO import StringIO
+
+    buf = StringIO()
+    df = pd.DataFrame({'x': [1, 2, 3, 4], 'y': [1, 0, 1, 0]}, index=range(4))  # Force int index; No RangeIndex in dask
+    df.info(buf=buf)
+    stdout_pd = buf.getvalue()
+
+    buf = StringIO()
+    ddf = dd.from_pandas(df, npartitions=4)
+    ddf.info(buf=buf, memory_usage=True)
+    stdout_da = buf.getvalue()
+    stdout_da = stdout_da.replace(str(type(ddf)), str(type(df)))
+
+    assert stdout_pd == stdout_da
```

Comment:
This is a great test.  It would be good to perform the same test on a few different dataframes, including an empty one, one without columns, etc..  Perhaps this could be extended with a for loop.